packages/loot-design/src/svg/v1/ChartBar.js

```diff
@@ -5,6 +5,7 @@ const SvgChartBar = props => (
     {...props}
     xmlns="http://www.w3.org/2000/svg"
     viewBox="0 0 20 20"
+    id={props.id}
```

Comment:
Did you do this manually? This code is auto-generated so it will get undone the next time we run the script. (And the `...props` above should take care of this, right?)

Reply:
Yes, it should. Good point. I'll look into it.

Reply:
Removed.